packages/pn-personafisica-webapp/src/component/Contacts/SpecialContacts.tsx

```diff
@@ -154,45 +171,66 @@ const SpecialContacts = ({ recipientId, legalAddresses, courtesyAddresses }: Pro
     initialValues,
     validateOnMount: true,
     validationSchema,
-    onSubmit: (values) => {
-      initValidation(
-        values.addressType,
-        values.s_pec || values.s_mail || internationalPhonePrefix + values.s_phone,
-        recipientId,
-        values.sender,
-        async (status: 'validated' | 'cancelled') => {
-          if (status === 'validated') {
-            // reset form
-            formik.resetForm();
-            await formik.validateForm();
+    onSubmit: (values: SpecialContacts) => {
+      if (values.addressType) {
+        initValidation(
+          values.addressType,
+          values.s_pec || values.s_mail || internationalPhonePrefix + values.s_phone,
+          recipientId,
+          values.sender.id,
+          async (status: 'validated' | 'cancelled') => {
+            if (status === 'validated') {
+              // reset form
+              formik.resetForm();
+              await formik.validateForm();
+            }
           }
-        }
-      );
+        );
+      }
     },
   });
 
+  const handleFilterAutocomplete = (options: Array<Party>, state: { inputValue: string }) => {
```

Comment:
Since you're using a setState declaration of inputValue (which you have to remane according to the comment above), state parameter is not needed. Just use the variable without state prefix.

Reply:
do it